apps/volunteer/shifts/talks.py

```diff
@@ -0,0 +1,83 @@
+from ..event_date import edt
+
+herald_pattern = [
+            {
+                "first": edt("fri", "09:00:00"),
+                "final": edt("fri", "21:00:00"),
+                "min": 1,
+                "max": 1,
+            } for d in ["fri", "sat", "sun"]
+        ]
+stage_pattern = [
+    {"first": edt(d, "10:00:00"),
+     "final": edt(d, "18:00:00"),
```

Comment:
Friday - Stage A is 09:30 to 20:00. Stage B/C should be 10:45 to 20:00
Saturday - All stages 09:45 to 20:00
Sunday - Stage A is 09:45 to 19:00. Stage B/C should be 09:45 to 18:30
(that's the current plan but subject to change by Jonty!)